.github/workflows/build.yml

```diff
@@ -292,9 +320,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  final_newline_lint:
+  check_final_newline:
```

Comment:
Do we really need to change the job id?

Reply:
It is inconsistent with all the other jobs